ostelco-core/Models/MyInfoDetails.swift

```diff
@@ -8,16 +8,32 @@
 
 import Foundation
 
-struct MyInfoAddress: Codable {
-    let country: String?
-    let unit: String?
-    let street: String?
-    let block: String?
-    let postal: String?
-    let floor: String?
-    let building: String?
+public struct MyInfoAddress: Codable {
```

Comment:
Any reason why we add public, is it good practice or required? (just out of curiosity)

Reply:
Yep, it's because it's being moved to Core. Core is a separate framework from the main application, and for anything to be seen outside the framework, it must be public.

The good news is with a `public let` since it's only created at instantiation, outside callers can't change it. That is *not* the case for non-computed `public var`s, so be aware of that.